inst/help/mlclassificationneuralnetwork.md

```diff
@@ -0,0 +1,85 @@
+
+### Assumptions
+- The target is a nominal or ordinal variable. 
+- The predictor variables consist of continuous variables.
+
+### Input 
+-------
+#### Assignment Box 
+- Target: In this box, the variable that needs to be predicted should be entered. 
+- Predictors: In this box the variables that provide information about the target variable should be entered. 
+
+#### Tables  
+- Confusion matrix: Displays a table that shows the observed classes against the predicted classes. Used to assess model accuracy.
+- Class proportions: Displays a table that shows the proportions of each class in the data set, training (and validaton), and test set.
+- Evaluation metrics: Shows commonly used classification evaluation metrics like precision, recall, the F1-score, support and AUC (area under the ROC curve).
+- Network weights: Shows the connections in the neural network together with their weights.
+
+#### Plots
+- Data split: Shows how the data is split into training (and validation), and test set.
+- Classification accuracy: Plots the number of nearest neighbors against the classification accuracy of the model. Accuracy is assessed for the training (and validation) set.
+- ROC curves: Displays ROC curves for each class predicted against all other classes.
+- Andrews curves: Is a way to visualize structure in high-dimensional data. Lines that cluster are observations that are more alike. 
+- Network structure: Creates a plot that visualizes the structure (nodes and edges) of the network.
+- Decision boundary matrix: Creates a *n* x *n* plot that visualizes how every observation would be classified if predicted through the current model. Boundaries between classes are visualized. Can only be made for numeric predictors.
+
+### Data Split Preferences
+#### Holdout Test Data
+- Sample *x*% of all data: Choose a percentage to randomly sample from your data to derive prediction error. Generates an internal indicator variable that indicates whether the observation is included (1) or excluded (0) from the test set.
+- Add generated indicator to data: Add the generated test set indicator from the option above to your data set. Requires a column name.
+- Test set indicator: Use an indicator variable to select data for the test set. This indicator should be a column in your data that consists of only 0 (excluded from the test set) and 1 (included in the test set). The data will then be split into a training (and validation if requested) set (0), and a test set (1) according to your indicator.
+
+#### Training and Validation Data
```

Comment:
I haven't read beyond here, but perhaps I can just edit this directly, and ask for your approval. Overall it's very good.

Reply:
You can

Reply:
Just remember that these things are equal for all ml analyses, so if anything changes it should change everywhere to keep consitent

Reply:
Okay good to know, then let's get going first and optimise later.